src/pudl/metadata/resources/eia930.py

```diff
@@ -155,16 +128,13 @@
         "create_database_schema": False,
     },
     "out_eia930__hourly_operations": {
-        "description": (
-            """EXPERIMENTAL / WORK-IN-PROGRESS, 2025-04-04.
+        "description": {
+            "additional_summary_text": "balancing authority net generation, interchange, and demand with imputed demand.",
+            "additional_details_text": """(EXPERIMENTAL / WORK-IN-PROGRESS, 2025-03-31)
 
-This adds imputed demand to the ``core_eia930__hourly_operations`` table where the
-original data was missing or anomalous.  Codes explaining why values have been imputed
-can be found in the ``core_pudl__codes_imputation_reasons`` table.
-
-This table is available in the nightly builds during development, but has not been fully
-vetted yet."""
```

Comment:
I don't feel great about removing this text. Is the idea that it's already covered by the EXPERIMENTAL header on additional details?

Reply:
Added it back!